API.Tests/Utilities/ClaimsPrincipalExtensionsTests.cs

```diff
@@ -71,18 +70,17 @@ public void ClaimsPrincipal_GetTokenType()
     }
 
     [Fact]
-    public void ClaimsPrincipal_GetRateLimitOverrides()
+    public void ClaimsPrincipal_GetRateLimitOverride()
     {
+        const int expected = 100;
```

Comment:
Constants generally are formatted with PascalCase or SCREAMING_SNAKE_CASE. Your call though since it's a local variable.